<launch>
   <group ns="pc_filter">
      <param name="xpassthrough/filter_limit_min" value="-0.5" />
      <param name="ypassthrough/filter_limit_min" value="-0.5" />
      <param name="zpassthrough/filter_limit_min" value="0.2" />
      <param name="xpassthrough/filter_limit_max" value="0.5" />
      <param name="ypassthrough/filter_limit_max" value="0.5" />
      <param name="zpassthrough/filter_limit_max" value="1.0" />
      <param name="target_color/r" value="50" />
      <param name="target_color/g" value="50" />
      <param name="target_color/b" value="50" />
      <param name="cluster/dist_threshold" value="0.05" />
      <param name="cluster/min_pts" value="8" />
      <param name="cluster/max_pts" value="1000" />
      
      <param name="observed_frame_id" value="camera_depth_optical_frame" />
      <param name="filtered_frame_id" value="camera_depth_optical_frame" />
      <param name="input_pc_topic" value="/filtered_pc" />
      <param name="output_pc_topic" value="/filtered_pc_after" />
      <param name="ugv_center_xy_topic" value="/ugv_center" />
      <param name="uav_big_tag_xy_topic" value="/uav_big_tag_xy" />
      <param name="uav_small_tag_xy_topic" value="/uav_small_tag_xy" />
      <param name="uav_tag_center_xy_topic" value="/uav_tag_center_xy" />
      <param name="uav_tag_range_topic" value="/uav_tag_dist" />
      
      <node name="rs_pc_filter" pkg="offboard_control" type="pc_filter_node" output="screen"/>
   </group>
</launch>
